util/helm/cmd.go

```diff
@@ -132,40 +220,125 @@ func writeToTmp(data []byte) (string, io.Closer, error) {
 	}), nil
 }
 
-func (c *Cmd) Fetch(repo, chartName, version, destination string, creds Creds) (string, error) {
-	args := []string{c.pullCommand, "--destination", destination}
+func (c *Cmd) Fetch(repoNamespace, repo, chartName, version, destination string, creds Creds) (string, error) {
+	output := ""
+	var err error
+	if c.ociSupported {
+		args := []string{"chart", "pull"}
+		repoUrl := fmt.Sprintf(repo + "/" + repoNamespace + "/" + chartName + ":" + version)
+		log.Infof("Helm OCI-based repository is %s", repoUrl)
```

Comment:
This should be `Debugf()` if required at all

Reply:
will update it